lib/main_common.pm

```diff
@@ -176,6 +205,19 @@ sub is_bridged_networking {
     return $ret;
 }
 
+sub is_livesystem {
+    return (check_var("FLAVOR", 'Rescue-CD') || get_var("LIVETEST"));
+}
+
+sub is_kde_live {
```

Comment:
shouldn't we keep everything which calculate version in version_utils ?

Reply:
After discussing with you in person I moved some functions into version_utils as well as utils but not this one because it is only used in this file here and also not exported.